dbt/models/proximity/proximity.dist_pin_to_stadium.sql

```diff
@@ -0,0 +1,76 @@
+-- CTAS to create a table of distance to the stadiums for each PIN
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['pin10'],
+        bucket_count=1
+    )
+}}
+
+-- Define stadium names and locations
+WITH stadiums AS (
+    SELECT
+        'Soldier Field' AS stadium_name,
+        ST_POINT(1179603, 1893122) AS stadium_xy
+    UNION ALL
+    SELECT
+        'Wintrust Arena' AS stadium_name,
+        ST_POINT(1178259, 1889955) AS stadium_xy
+    UNION ALL
+    SELECT
+        'Guaranteed Rate Field' AS stadium_name,
+        ST_POINT(1174703, 1881406) AS stadium_xy
+    UNION ALL
+    SELECT
+        'Wrigley Field' AS stadium_name,
+        ST_POINT(1168634, 1924426) AS stadium_xy
+    UNION ALL
+    SELECT
+        'United Center' AS stadium_name,
+        ST_POINT(1163584, 1899780) AS stadium_xy
+    UNION ALL
+    SELECT
+        'UIC Pavilion' AS stadium_name,
+        ST_POINT(1168816, 1897726) AS stadium_xy
```

Comment:
I know I missed the boat on this PR, but @Damonamajor can we actually move this into a dbt seed? Something like `spatial.stadium`, with the columns `name`, `year`, `lon`, and `lat`. There are example seeds in `dbt/seeds` for you to reference.